### DISCUSSION TITLE
RoderickShao_notebook/%E9%80%9A%E8%AF%86%E5%9F%BA%E7%A1%80%E8%AF%BE/%E6%A6%82%E7%8E%87%E8%AE%BA%E4%B8%8E%E6%95%B0%E7%90%86%E7%BB%9F%E8%AE%A1/%E7%AC%AC%E4%B8%83%E7%AB%A0%20%E5%8F%82%E6%95%B0%E4%BC%B0%E8%AE%A1/

### DISCUSSION THREAD

**Discussion by giscus** [RoderickShao/RoderickShao_notebook#15] [Announcements]
# RoderickShao_notebook/%E9%80%9A%E8%AF%86%E5%9F%BA%E7%A1%80%E8%AF%BE/%E6%A6%82%E7%8E%87%E8%AE%BA%E4%B8%8E%E6%95%B0%E7%90%86%E7%BB%9F%E8%AE%A1/%E7%AC%AC%E4%B8%83%E7%AB%A0%20%E5%8F%82%E6%95%B0%E4%BC%B0%E8%AE%A1/

RoderickShao's NoteBook

https://roderickshao.github.io/RoderickShao_notebook/%E9%80%9A%E8%AF%86%E5%9F%BA%E7%A1%80%E8%AF%BE/%E6%A6%82%E7%8E%87%E8%AE%BA%E4%B8%8E%E6%95%B0%E7%90%86%E7%BB%9F%E8%AE%A1/%E7%AC%AC%E4%B8%83%E7%AB%A0%20%E5%8F%82%E6%95%B0%E4%BC%B0%E8%AE%A1/

**Comment by Provence-24** [comment]
第七章内容尝试梳理：第六章研究完样本后，第七章尝试引入总体的参数，通过样本的信息去预测总体。参数估计分为点估计和区间估计。点估计分为矩法和最大似然估计法。前者的方法是：用参数表示总体的矩，用矩反解参数，用样本矩代替矩，从而得到的参数的矩估计量，进而有矩估计值。最大似然估计法，引入似然方程，一般要取对数，最后求导使其取到极值，找到对应的参数。这里面会有一个max的例子，需要掌握。点估计之后，需要对参数估计的效果进行评价，引入四个量：无偏性、有效性、均方误差、相合性。与第四章的均值方差结合，也与第五章的依概率收敛结合，体系十分完善。区间估计比较死，对应哪一种类型，直接往里面套，都是双边的，正态和t分布都是对称的，但是F不是对称的。
